src/ledger/insert_shred.zig

```diff
@@ -1861,77 +1861,72 @@ fn newlinesToSpaces(comptime fmt: []const u8) [fmt.len]u8 {
 
 const test_shreds = @import("test_shreds.zig");
 const comptimePrint = std.fmt.comptimePrint;
+const TestDB = ledger.tests.TestDB("insert_shred");
```

Comment:
not used?

Reply:
58ad7851e9220723184c5eac0ec394dff0fcccc0